gitpod-io/website
# Requests & Responses examples for Public API

## Description

cURL Requests & Responses examples for Public API

## Related Issue(s)



Fixes #3191

## How to test

- Go To [**`/docs/references/gitpod-public-api`**](https://deploy-preview-3196--gitpod-kumquat.netlify.app/docs/references/gitpod-public-api) & see :eyes:

## Release Notes


```release-note
Docs: Public API _(Alpha)_
```


<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/3196"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
